server/events/event_parser_test.go

```diff
@@ -1235,6 +1235,22 @@ func TestParseAzureDevopsRepo(t *testing.T) {
 		},
 	}, r)
 
+	// this should be successful
+	repo = ADRepo
+	repo.WebURL = azuredevops.String("https://owner.visualstudio.com/project/_git/repo")
```

Comment:
Can we have another test without visualstudio in the url to check both the if block and else block?

Reply:
No problem. Have added 🙂